src/lib/blockly/generators/Spatext.ts

```diff
@@ -1,7 +1,18 @@
 import Blockly from ".."
-import { formulas } from "../types"
 
 class SpatextGenerator extends Blockly.CodeGenerator {
+    _outputType = (block:Blockly.Block|null):"math"|"language"|null => {
```

Comment:
added a helper method to decide if the SpaTeXt being spit out by the code generator was "language" or "math". In the latter case, I need to be careful about composing math inside language vs math inside math...